routes/web.php

```diff
@@ -23,5 +23,7 @@
     return File::get(public_path() . '/docs/assets/js/*.js');
 });
 
-Route::get('/{any}', 'SpaController@index')->where('any', '.*');
+Route::resource('profile', 'ProfileController');
+
+//Route::get('/{any}', 'SpaController@index')->where('any', '.*');
```

Comment:
Why was this commented out?

Reply:
Oh it wasn't supposed to be. I'll fix it